src/pages/_app.tsx

```diff
@@ -47,6 +50,18 @@ class MyApp extends App<Props> {
     super.componentDidCatch(error, errorInfo);
   }
 
+  // next focuses body between route changes if it has tabIndex=-1
```

Comment:
Seem to work when I tested. Just wonder if there would be a chance to write a test for this (at least to check the tabindex is in it's place). But it might be quite hard thing to do on component tests and too corner case for  browser-tests.

Reply:
Hmm one thing is that we should do the same change on palvelutarjotin-admin as well? At least according to the audit report. I dont know if there is separate ticket for it but It could be done now soon.

Reply:
Yeah noticed that also today.. This ticket was only for Teacher UI. Lets add another ticket if it is missing from the board

Reply:
We can easily add one test for the focus stuff in browser test later